oeo-physical.omn

```diff
@@ -1538,7 +1549,9 @@ Class: Hydrogen
         <http://purl.obolibrary.org/obo/IAO_0000119> "https://wiki.openmod-initiative.org/wiki/Hydrogen"@en
     
     SubClassOf: 
-        EnergyStorage
+        EnergyStorage,
```

Comment:
Classification of 'Hydrogen' and 'EnergyStorage' needs to be discussed. For the moment I'd propose to make Hydrogen a subclass of 'PortionOfMatter'.

Reply:
I opend an issue for EnergyStorage #209

Reply:
I'll leave that for another pull request and your issue